go/integTest/batch_test.go

```diff
@@ -122,12 +122,34 @@ func CreateStringTest(batch *pipeline.ClusterBatch, isAtomic bool) BatchTestData
 	return BatchTestData{CommandTestData: testData, TestName: "String commands"}
 }
 
+func CreateGenericClusterCommandTests(batch *pipeline.ClusterBatch, isAtomic bool) BatchTestData {
+	testData := make([]CommandTestData, 0)
+	key := "{key}-1-" + uuid.NewString()
+
+	// Start clean
+	batch.FlushAll()
+	testData = append(testData, CommandTestData{ExpectedResponse: "OK", TestName: "CustomCommand: FlushAll()"})
+
+	batch.CustomCommand([]string{"SET", key, "1"})
+	testData = append(testData, CommandTestData{ExpectedResponse: "OK", TestName: "CustomCommand: Set(key, 1)"})
+	batch.Get(key)
+	testData = append(testData, CommandTestData{ExpectedResponse: "1", TestName: "Get(key1)"})
+
+	batch.RandomKey()
+	testData = append(testData, CommandTestData{ExpectedResponse: key, TestName: "RandomKey()"})
+
+	// TODO: add Move in separate standalone batch tests
+
+	return BatchTestData{CommandTestData: testData, TestName: "Generic Cluster commands"}
```

Comment:
```suggestion
	return BatchTestData{CommandTestData: testData, TestName: "Generic commands"}
```
This test executed on cluster and standalone clients both

Reply:
ah! nice catch